atom/browser/api/atom_api_web_contents.h

```diff
@@ -6,13 +6,17 @@
 #define ATOM_BROWSER_API_ATOM_API_WEB_CONTENTS_H_
 
 #include <string>
+#include <vector>
+#include <set>
```

Comment:
Yeah its because of that, but i decided to have the standard headers in one place in `api_web_contents.h` , should i move it ?